app/assets/v2/js/pages/bulk_payout.js

```diff
@@ -166,6 +166,10 @@ $(document).ready(function($) {
       _alert('You do not have any transactions to payout.  Please add payees to the form.', 'error');
       return;
     }
+    if ($('.username').val().trim() == '' || $('.username').val() === null) {
```

Comment:
Firstly, `$('.username')` isn't an input field. It is a `div` inside a table row. So` .val()` should always return empty string. So it should use `.text()` instead of `.val()`.

Secondly, checking for empty string and null isn't enough. You also need to check if the string contains only '@', because that is what it defaults to.

Thirdly, you need to check in all rows and if there is a row with a percentage amount and no username, it should give error. A scenario as shown in the below screenshot.
<img width="322" alt="screen shot 2018-09-05 at 1 02 23 pm" src="https://user-images.githubusercontent.com/9530293/45077874-0a941180-b10c-11e8-8636-bba199adb0f4.png">

Fourth, please stick to `===`. I see you used `==` in one part of condition and `===` in another. :)

Reply:
@SaptakS thanks for the feedback, will fix.